microsoft-edge/webview2/how-to/prerelease-testing.md

```diff
@@ -0,0 +1,246 @@
+
+```
+WebView2 Runtime (Stable) ->  Edge Beta -> Edge Dev -> Edge Canary
+```
+
+To do prerelease testing, reverse the channel search order, so that the loader looks for binaries in order from least-stable (Canary) to most-stable:
+
+```
+Edge Canary -> Edge Dev -> Edge Beta -> WebView2 Runtime (Stable)
+```
+
+The channel search kind can be set per app through a registry key, environment variable, API, or group policy.
+
+The value `1` indicates reversed search order; that is, from least-stable (Edge Canary) to most-stable.
+
+
+#### [Registry key](#tab/registry-key)
+
+```reg
+REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\ChannelSearchKind /v WebView2APISample.exe /t REG_DWORD /d 1
+```
+
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.
+
+
+#### [Environment variable](#tab/environment-variable)
+
+Name: `WEBVIEW2_CHANNEL_SEARCH_KIND`  
+Value: `1`
+
+
+#### [API](#tab/api)
+
+By default, the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property is `CoreWebView2ChannelSearchKind.MostStable` (an enum value).  Instead, reverse the search order by setting the `CoreWebView2EnvironmentOptions.ChannelSearchKind` property to `CoreWebView2ChannelSearchKind.LeastStable`.
+
+.NET:
+* [CoreWebView2EnvironmentOptions.ChannelSearchKind Property](/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.channelsearchkind)
+* [CoreWebView2ChannelSearchKind Enum](/dotnet/api/microsoft.web.webview2.core.corewebview2channelsearchkind)
+
+WinRT:
+* [CoreWebView2EnvironmentOptions.ChannelSearchKind Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions#channelsearchkind)
+* [CoreWebView2ChannelSearchKind Enum](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2channelsearchkind)
+
+Win32:
+* [ICoreWebView2ExperimentalEnvironmentOptions::get_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions#get_channelsearchkind)
+* [ICoreWebView2ExperimentalEnvironmentOptions::put_ChannelSearchKind](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalenvironmentoptions#put_channelsearchkind)
+* [COREWEBVIEW2_CHANNEL_SEARCH_KIND enum](/microsoft-edge/webview2/reference/win32/webview2experimental-idl#corewebview2_channel_search_kind)
+
+
+#### [Group policy](#tab/group-policy)
+
+Set the `ChannelSearchKind` policy.
+
+Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`  
+Value: `1`
+
+Do either of the following:
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  Click any of the three **Download Windows Policy** links.<!-- todo: no "Edge policy" download link there -->
+* Use the built-in policy on Intune.<!-- todo: link to more info -->
```

Comment:
What's a URL to get more info about "Use the built-in policy on Intune"?  @victorhuangwq

Reply:
I don't believe we have any WebView2-specific steps, but it's basically the same as the Edge steps:
https://learn.microsoft.com/en-us/deployedge/configure-edge-with-intune

Only difference is that the category is Microsoft Edge WebView2 instead of Microsoft Edge

Reply:
@vickiez @victorhuangwq - my impression is that both articles are relevant, and a little different for different contexts, and user might want to see both articles, so keep both links.  Closing.